hello-world/build.gradle

```diff
@@ -17,10 +17,10 @@ subprojects {
 
     repositories {
         // This repo is only for the master branch, and must be not present in release/* branches.
-        maven {
-            name = "R3 artifactory"
-            url = "https://software.r3.com/artifactory/conclave-maven"
-        }
+//        maven {
+//            name = "R3 artifactory"
+//            url = "https://software.r3.com/artifactory/conclave-maven"
+//        }
```

Comment:
I assume keeping those comments is deliberate?

Reply:
Yes, just in case it needs to be put back (of course very unlikely)